tests/unit/services/StatementBuilder.unitTest.ts

```diff
@@ -0,0 +1,56 @@
+import StatementBuilder from "../../../src/services/StatementBuilder";
+
+describe('StatementBuilder', (): void => {
```

Comment:
I was wondering if we wanted to add some unit tests for the getters and setters too?

Reply:
As agreed on the call, this is a bit too fine-grained for this project, so resolving. Could always be added in future